ntrans_gui.py

```diff
@@ -190,7 +202,10 @@ def generate_ntrans_dictionary(self) -> None:
         # TODO: Check filepath for validity
 
         # Calls logic in ntrans.py
-        ntrans.read_ngram_files(self.user_choices)
+        thread = threading.Thread(
+            target=ntrans.read_ngram_files, args=[self.user_choices]
```

Comment:
A few notes:
- [ ] `user_choices` should be a local variable, not an attribute of `self`. As usual, every attribute of `self` should be defined in `__init__`. 
- [ ] Disable the button when you make a thread, so user can't click the "Generate N-Trans Dictionary" button twice and have two threads running at the same time
- [ ] Progress bar might be nice